test/e2e/e2e.go

```diff
@@ -158,6 +158,9 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 	// Reference federation test to make the import valid.
 	federationtest.FederationSuite = commontest.FederationE2E
 
+	// Run test suite init actions (if any) added. Called only once for test suite during setup.
+	framework.RunSuiteInitActions()
```

Comment:
if there are less abstract ways to accomplish setup in the federation case, I'd prefer that to a global before/after registration mechanism